CMakeLists.txt

```diff
@@ -8,8 +8,8 @@ project(ignition-plugin2 VERSION 2.0.0)
 #============================================================================
 # Find ignition-cmake
 #============================================================================
-find_package(ignition-cmake2 2.8.0 REQUIRED)
-set(IGN_CMAKE_VER ${ignition-cmake2_VERSION_MAJOR})
+find_package(ignition-cmake3 3.0.0 REQUIRED)
```

Comment:
As we learned in `ign-utils`, it's probably better to remove the version here

Reply:
https://github.com/ignitionrobotics/ign-plugin/pull/74/commits/e60bc5da114bbdea6b3eebf2f1e5868ac927cc25